libs/sdk-ui-kit/package.json

```diff
@@ -46,7 +46,8 @@
         "react-codemirror2": "^7.2.0",
         "react-day-picker": "7.3.0",
         "react-intl": "^3.6.0",
-        "tslib": "^2.0.0"
+        "tslib": "^2.0.0",
+        "react-native-listener": "~1.1.0"
```

Comment:
I see this was copied from goodstrap, but maybe this might be a good time to revisit if this is really necessary here. Do you know why this is needed?

Reply:
needed